docs/courses/intro_to_gui.rst

```diff
@@ -51,4 +51,213 @@ or in Python
+
+Make sure there is a check mark in the "Specify Strategy" box.
+
+Select "area".
+
+Click on "Specify Strategy" to clear the check mark.
+
+Make sure that area is 100 um^2. 
+
+6. 
+    **Click on "Continuous Create" box so that a check mark appears in it.**
+
+        .. image:: img/continuouscreate.png
+            :align: center
+
+Congratulations! You now have a model cell with a soma that has surface area = 100 um^2. 
+
+**Save this to a "session file" so you can recreate it whenever you like.**
+
+:menuselection:`NEURON Main Menu --> File --> save session`
+
+Using the computational model
+++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+You'll need these items:
+
+- controls for running simulations
+- a graph that shows somatic membrane potential v. time
+- a current clamp to inject a current pulse into the model cell
+- a graph that shows the stimulus current vs. time
+
+*Be sure to save your work to a session file as you go along!*
+
+**Make a RunControl panel for launching simulations.**
+
+:menuselection:`Neuron Main Menu (NMM) --> Tools --> RunControl`
+
+    .. image:: img/runcontrol.png
+        :align: center
+
+**Make a graph that shows somatic membrane potential vs. time**
+
+:menuselection:`NMM --> Graph --> Voltage axis`
+
+    .. image:: img/voltageaxis.png
+        :align: center
+
+Now run a simulation by clicking on RunControl's "Init & Run". What happened?
+
+**Make a current clamp.**
+
+:menuselection:`NMM --> Tools --> Point Processes --> Managers --> Point Manager`
+
+    .. image:: img/pointmanager.png
+        :align: center
+
+Make this an IClamp.
+
+    :menuselection:`PointProcessManager --> SelectPointProcess --> IClamp`
+
+Show the IClamp's Parameters panel
+
+    :menuselection:`PPM --> Show --> Parameters`
+
+Make it deliver a 1 nA x 1 ms current pulse that starts at 1 ms.
+
+    del (ms) = 1
+
+    dur (ms) = 1
+
+    amp (nA) = 1
+
+**Make a graph that shows the stimulus current vs. time.**
+
+:menuselection:`NMM --> Graph --> Current axis`
+
+    .. image:: img/currentaxis.png
+        :align: center
+
+Make it show the IClamp's i
+
+- Click on the graph's menu button (left upper corner of its canvas) and select "Plot what?"
+- Plot what? / Show / Objects
+- Select IClamp (left panel)
+- Select 0 (middle panel)
+- Select i (middle panel)
+- Plot what?'s edit field should now contain IClamp[0].i
+- Click on Accept
+
+Run a simulation.
+
+Too big? Divide IClamp.amp by 10 and try again.
+
+    .. image:: img/iclampamptryagain.png
+        :align: center
+
+
+Exercises
+~~~~~~~~~~
+
+1. 
+    **Run a simulation.**
+
+Double the duration and halve the amplitude of the injected current (injected charge is constant).
+
+Notes:
+
+- :menuselection:`Select Show --> Parameters in the PointProcessManager window to view the field editors.`
+- Multiplying a field editor value by 2 can be done by typing ``*2`` <return> at the end of the value. Divide by typing /2 <return>.
```

Comment:
if we're putting ``*2`` in backticks, we should do the same for the inverse ``/2``